src/views/ConfigurationLogsView.vue

```diff
@@ -1,11 +1,14 @@
 <template>
   <BaseConfigurationView>
-    <template #title>Logs configuration</template>
+    <template #title>Logs configuration for {{ currentView }}</template>
     <template #content>
       <h1 class="text-lg font-bold text-slate-600">Variables to be show in the overlay subtitles log:</h1>
+      <span class="text-sm text-slate-400 w-[50%] text-center">
+        Available variables are independant for each view.
+      </span>
       <div class="flex w-[60%] flex-wrap">
         <v-checkbox
-          v-for="variable in DatalogVariable"
+          v-for="variable in loggedVariables"
```

Comment:
It would be nice to have the items to be organized alphabetically, otherwise it's kind of hard to predict where it is.